cpp/tensorrt_llm/cortex.tensorrt-llm/src/tensorrt-llm_engine.h

```diff
@@ -119,7 +190,8 @@ class TensorrtllmEngine : public EngineI {
   void GetModelStatus(
       std::shared_ptr<Json::Value> json_body,
       std::function<void(Json::Value&&, Json::Value&&)>&& callback) final;
-
+  virtual std::vector<int> encode_header_llama3(const std::string& role);
```

Comment:
should rename the function to `EncodeHeaderLlama3`, similar for `EncodeMessageLlama3`